### PR TITLE
add license info of tv4, cc #5500

### PR DESCRIPTION
**Pull request by clairetsai818**
https://github.com/geraintluff/tv4#license

### CODE REVIEW

**Comment by clairetsai818** [comment]
@PeterDaveHello  Please help me review it, thanks.

File: package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "jslint": "^0.10.1",
     "vows-si": "^0.7.1"
   },
+  "license": ["MIT", "public domain"],
```
Comment (PeterDaveHello) [inline package.json]: as "public domain" is not a license, we could just use MIT here.

**Comment by clairetsai818** [comment]
Removed "Public domain".
@PeterDaveHello Please help me review it, thanks.

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

**Pull request merged**